omod/src/main/webapp/resources/scripts/components/LabTrackingDataService.js

```diff
@@ -521,7 +545,7 @@ angular.module("labTrackingDataService", [])
                 return ensureActiveVisit(labTrackingOrder.patient.value, labTrackingOrder.location.value, labTrackingOrder.visit)
                     .then(function (res) {
                         if (_self.isOk(res)) {
-                            return _self.createOrderEncounter(labTrackingOrder);
```

Comment:
Probably makes sense to deprecate or delete the "createOrderEncounter" function and any other related functionality, so we don't worry about maintaining it (or trying to remind ourselves what it does!) in the future!